examples/AzureFunctionSASTokenVendor/GetSasToken/__init__.py

```diff
@@ -0,0 +1,50 @@
+import datetime, json, os
+
+import azure.functions as func
+import azure.storage.blob as blob
+
+account_name = os.environ['AZURE_STORAGE_ACCOUNT_NAME']
+account_key = os.environ['AZURE_STORAGE_ACCOUNT_KEY']
+container_name = os.environ['AZURE_STORAGE_CONTAINER_NAME']
+
+def main(req: func.HttpRequest) -> func.HttpResponse:

```

Comment:
So this is the only Python-specific code needed for the Azure Function? @adrianhall any feedback on this file?

Reply:
Yep, everything else in this was generated by the VSCode extension when I used it to create a new Python-based Azure Function project. I also had to update requirements.txt and and I force-added the local.settings.json file (normally excluded by gitignore) to include an example of the three environment variables the function needs to access.